src/Database/DBA.php

```diff
@@ -49,6 +50,7 @@ public static function connect($serveraddr, $user, $pass, $db, $charset = null)
 		self::$db_name = $db;
 		self::$db_charset = $charset;
 
+		$port = false;
```

Comment:
Why not initialize with `0`?

Reply:
Hmm, okay. The documentation says `int $port = ini_get("mysqli.default_port")`. Then best would be to take that then?

Reply:
Then I can save the `if/else` block.

Reply:
No because it’s only for the `mysqli` constructor. Same problem as with 3306 before.

Reply:
So `0` is the proper choice here?

Reply:
Yes, with the conditional for `mysqli`.